src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -30,13 +30,20 @@
 					</option>
 				</kv-select>
 			</div>
-			<div v-if="!removeGenderDropdown && !withCategories" class="tw-overflow-x-auto tw-pb-1">
+			<div v-if="!removeGenderDropdown && !withCategories" class="tw-flex tw-gap-1 tw-overflow-x-auto tw-pb-1">
 				<filter-pills
 					:filters-loaded="filtersLoaded"
 					:options="filterOptions.gender"
 					:selected-values="selectedGenders"
 					@update-values="updateGenders($event)"
 				/>
+				<div v-if="!filtersLoaded" class="tw-flex tw-gap-1 placeholder">
+					<kv-loading-placeholder
+						style="height: 2.75rem; width: 100px;"
```

Comment:
Looks like the loading placeholders are 1px shorter, and the `height` can be removed here to fix that.

Reply:
Done.